test/uk/gov/hmrc/helptosavefrontend/connectors/SessionCacheConnectorImplSpec.scala

```diff
@@ -29,9 +30,12 @@ import uk.gov.hmrc.http.cache.client.CacheMap
 import uk.gov.hmrc.http.{HeaderCarrier, HttpReads}
 
 import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
 
 class SessionCacheConnectorImplSpec extends TestSupport with ScalaFutures with GeneratorDrivenPropertyChecks {
 
+  implicit val config: PatienceConfig = PatienceConfig().copy(timeout = Span.convertDurationToSpan(10.seconds))
```

Comment:
I guess you can simply do `timeout = scaled(10.seconds)` without needing to use `Span.convertDurationToSpan`